src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -602,8 +618,6 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
   caption: [UC6.2 Filtro per tipologia #glossary("sensore") su tabella]
 )
```

Comment:
Questo vale solo per dati grezzi, e non per tutte le tabelle o sbaglio?

Reply:
corretto, non credo sia un problema visto che abbiamo Il #glossary("pannello") offre la funzionalità di filtro dei dati tramite apposita icona o pulsante. nelle precondizioni; in alternativa, come lo dovei esplicitare?

Reply:
si ma è anche l'unica tabella che riporta il tipo del sensore se non sbaglio, quindi credo che in realtà vada bene, e ci fossero altre tabelle con quel campo anche quelle dovrebbero poter essere ornate (?)